src/main/java/com/datastax/cdm/schema/CqlTable.java

```diff
@@ -54,99 +54,99 @@
 import com.datastax.oss.driver.api.core.type.codec.registry.MutableCodecRegistry;
 
 public class CqlTable extends BaseTable {
-    public Logger logger = LoggerFactory.getLogger(this.getClass().getName());
-    boolean logDebug = logger.isDebugEnabled();
-    boolean logTrace = logger.isTraceEnabled();
-
-    private final CqlSession cqlSession;
-    private boolean hasRandomPartitioner;
-    private final List<String> partitionKeyNames;
-    private final List<String> pkNames;
-    private final  List<Class> pkClasses;
-    private final List<Integer> pkIndexes;
-    private boolean isCounterTable;
-    private final ConsistencyLevel readConsistencyLevel;
-    private final ConsistencyLevel writeConsistencyLevel;
-
-    private List<ColumnMetadata> cqlPartitionKey;
-    private List<ColumnMetadata> cqlPrimaryKey;
-    private List<ColumnMetadata> cqlAllColumns;
-    private Map<String,DataType> columnNameToCqlTypeMap;
-    private final List<Class> bindClasses;
-    private List<String> writetimeTTLColumns;
-
-    private CqlTable otherCqlTable;
-    private List<Integer> correspondingIndexes;
-    private final List<Integer> counterIndexes;
-    protected Map<Featureset, Feature> featureMap;
+	public Logger logger = LoggerFactory.getLogger(this.getClass().getName());
```

Comment:
Can you make this whole file change be smaller - it is difficult to understand what has actually changed in this PR?